src/borg/testsuite/archiver.py

```diff
@@ -932,16 +932,20 @@ def _create_test_keep_tagged(self):
         self.create_regular_file('taggedall/%s' % CACHE_TAG_NAME,
                                  contents=CACHE_TAG_CONTENTS + b' extra stuff')
         self.create_regular_file('taggedall/file4', size=1024)
+        self.create_regular_file('taggeddir/.NOBACKUP2/file5', size=1024)
+        self.create_regular_file('taggeddir/file6', size=1024)
```

Comment:
as we now consider tags no matter whether they are files or dirs, maybe just modify line925 to cover the dirs case (and drop "taggeddir")?

tagged1..n dirs were meant to cover the n different ways to tag.

taggedall was meant to have all tagging methods present at once, maybe also modify that accordingly.